pipelinedp4j/main/com/google/privacy/differentialprivacy/pipelinedp4j/spark/SparkTable.kt

```diff
@@ -149,7 +167,40 @@ class SparkTable<K, V>(val data: Dataset<Tuple2<K, V>>,
         return filterKeys(stageName) {k -> k in allowedKeysHashSet}
     }
 
+    /**
+     * Randomly samples values per key. The output table will contain same keys as this table, each key will appear
```

Comment:
nit: let's keep first line short, just "Randomly samples values per key". And then with new paragraph continue.